src/app/shared/constants.ts

```diff
@@ -1,4 +1,6 @@
 export const url = {
   loginUrl: 'http://localhost:8080/login',
   registerUrl: 'http://localhost:8080/register',
+  restaurantIdUrl: 'http://localhost:8080/restaurants/add',
```

Comment:
By dividing all restaurant urls in separate properties we might make a mess here.
A better approach would be to have one `restaurantsUrl: 'http://localhost:8080/restaurants'` here, and just concatenate `add`, `id` etc. later in the service where we fetch data.
For example: `this.http.post<HttpResponse>(`${this.restaurantUrl}/add`, body)`